tools/azsdk-cli/Azure.Sdk.Tools.Cli/Services/Languages/JavaScriptLanguageSpecificChecks.cs

```diff
@@ -67,4 +69,77 @@ public async Task<CLICheckResponse> UpdateSnippetsAsync(string packagePath, Canc
             return new CLICheckResponse(1, "", $"Error updating snippets: {ex.Message}");
         }
     }
-}
+
+    public async Task<CLICheckResponse> LintCodeAsync(string packagePath, bool fix = false, CancellationToken cancellationToken = default)
+    {
+        try
+        {
+            var subcommand = fix ? "lint:fix" : "lint";
+            _logger.LogInformation($"Running 'pnpm run {subcommand}' in {packagePath}");
+
+            var result = await _processHelper.Run(new(
+                    "pnpm",
+                    ["run", subcommand],
+                    "pnpm.cmd",
+                    ["run", subcommand],
+                    workingDirectory: packagePath
+                ),
+                cancellationToken
+            );
+
+            if (result.ExitCode != 0)
+            {
+                _logger.LogError($"'pnpm run {subcommand}' failed with exit code {result.ExitCode}");
+
+                var nextSteps = fix ? "Review the linting errors and fix them manually." : "Run this tool in fix mode to automatically fix some of the errors.";
```

Comment:
Am I understanding this correctly - so if they have chosen to fix the errors and they aren't fixed then it tells them to re-run with -fix. what if they are in mcp and havent passed through fix bc they dont know it exists?